https://bugs.gentoo.org/396269
Install OpenDocument style files.

Makefile patch from upstream git repo,
commit e514982ae126c058b965dd00393b4e68e5f8522d by Dave Marquardt.

--- org-7.8.02-orig/Makefile
+++ org-7.8.02/Makefile
@@ -19,6 +19,12 @@
 # Where local lisp files go.
 lispdir   = $(prefix)/share/emacs/site-lisp
 
+# Where etc files go
+etcdir    = $(lispdir)/../etc
+
+# Where style files go
+stylesdir = $(etcdir)/styles
+
 # Where info files go.
 infodir = $(prefix)/share/info
 
@@ -175,6 +181,7 @@
 CARDFILES   = doc/orgcard.tex doc/orgcard.pdf doc/orgcard_letter.pdf
 TEXIFILES   = doc/org.texi
 INFOFILES   = doc/org
+STYLESFILES = etc/styles/OrgOdtContentTemplate.xml etc/styles/OrgOdtStyles.xml
 
 # Package Manager (ELPA)
 PKG_TAG = $(shell date +%Y%m%d)
@@ -211,7 +218,7 @@
 
 compile: $(ELCFILES0) $(ELCBFILES)
 
-install: install-lisp
+install: install-lisp install-etc
 
 doc: doc/org.html doc/org.pdf doc/orgcard.pdf doc/orgcard_letter.pdf doc/orgguide.pdf doc/orgcard.txt
 
@@ -231,6 +238,10 @@
 	$(CP) $(INFOFILES) $(infodir)
 	$(INSTALL_INFO) --infodir=$(infodir) $(INFOFILES)
 
+install-etc: $(STYLESFILES)
+	if [ ! -d $(stylesdir) ]; then $(MKDIR) $(stylesdir); else true; fi ;
+	$(CP) $(STYLESFILES) $(stylesdir)
+
 autoloads: lisp/org-install.el
 
 lisp/org-install.el: $(LISPFILES0) Makefile
--- org-7.8.02-orig/lisp/org-odt.el
+++ org-7.8.02/lisp/org-odt.el
@@ -76,7 +76,7 @@
   "Regular expressions for special string conversion.")
 
 (defconst org-odt-lib-dir (file-name-directory load-file-name))
-(defconst org-odt-styles-dir
+(defvar org-odt-styles-dir
   (let* ((styles-dir1 (expand-file-name "../etc/styles/" org-odt-lib-dir))
 	 (styles-dir2 (expand-file-name "./etc/styles/" org-odt-lib-dir))
 	 (styles-dir
